src/index.ts

```diff
@@ -44,23 +43,15 @@ const googleApplicationCredentials = getInput("firebaseServiceAccount", {
   required: true,
 });
 const configuredChannelId = getInput("channelId");
-const isProductionDeploy = configuredChannelId === "live";
 const token = process.env.GITHUB_TOKEN || getInput("repoToken");
 const octokit = token ? getOctokit(token) : undefined;
 const entryPoint = getInput("entryPoint");
 const target = getInput("target");
 const firebaseToolsVersion = getInput("firebaseToolsVersion");
 
 async function run() {
-  const isPullRequest = !!context.payload.pull_request;
-
-  let finish = (details: Object) => console.log(details);
-  if (token && isPullRequest) {
-    finish = await createCheck(octokit, context);
-  }
```

Comment:
I also prefer to avoid duplications. However, the changes in code are already done. Please, merge #345 and I will rebase my code to accommodate its changes. This way if that PR does not do smth or, in opposite, do some extras, I am able to see it.

Reply:
rebased and merged conflict changes